rlp-derive/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "rlp_derive"
```

Comment:
Do we want to change the name to `rlp-derive`? (now or never)

Reply:
Please!

Reply:
YESS!